### ISSUE TITLE
iso image

### ISSUE THREAD

**Issue by arsenieciprian** [voiceboys/sbcOS#30]
hi theare. from where i can take iso image ?

**Comment by jailbird777** [comment]
You can easily build your own customized ISO using the scripts provided.

However, I just spun a stock up one for you to test with: https://fdf.net/sbc-x86_64.iso

**Comment by airsay** [comment]
@jailbird777 apologies for resurrecting a necro. Do you by chance still have the iso? Or are you able to help with a step-by-step guide to building the iso on a Debian/Ubuntu VM on Proxmox?

**Comment by lmangani** [comment]
@airsay just clone the repository, go to SbcOS and run:
```
./build_rootfs.sh
```

**Comment by jailbird777** [comment]
It looks like it got removed since it was so old, I'll work on respinning it, although @lmangani 's instructions are basically all you need. Just make sure you update IPs everywhere first.

I had a PR open to refresh everything again, but I put it on hold since I wanted to rewrite the example iptables rules to nftables (I already switched my local copy over) and I also had issues with EFI booting the ISO under bhyve, so I rewrote that code. There's a few other loose ends I want to tidy up.

My end goal is to stop shipping binary packages in the repo (except the static initramfs tools) and have the build script just download everything. The biggest gotcha is how the handle the rtpengine kernel module, since the kernel that you're running under and the kernel that gets installed are probably going to be two different things completely. (eg, I'm experimenting with the RT kernel on my SBC at home).

**Comment by adubovikov** [comment]
guys, everything migrated to alpine os, tomorrow I will push all files and changes

**Comment by jailbird777** [comment]
Awesome! I knew that was the plan, it must have been a lot of work!

**Comment by airsay** [comment]
> guys, everything migrated to alpine os, tomorrow I will push all files and changes

That would be great! So I’m assuming I just need to spin up an Alpine VM and follow @lmangani ’s advice to run ./build_rootfs.sh. 

This is going to be for a home lab with the PBX and the SBC behind a NAT with static IP on the public facing router.

**Comment by airsay** [comment]
>The biggest gotcha is how the handle the rtpengine kernel module, since the kernel that you're running under and the kernel that gets installed are probably going to be two different things completely. (eg, I'm experimenting with the RT kernel on my SBC at home).

I’ve been stung so many times with installing RTPEngine that my default process now is to install linux-headers-$(uname -r) before any installation that require RTPEngine. Thanks for the heads up. Would still be interested in some guide on what IP addresses need updating

**Comment by arsenieciprian** [comment]
please add an iso somewere

**Comment by airsay** [comment]
> guys, everything migrated to alpine os, tomorrow I will push all files and changes

Did this ever happen?

**Comment by adubovikov** [comment]
yes, an year ago - https://github.com/voiceboys/sbcOS/tree/version_2.0

**Comment by airsay** [comment]
> yes, an year ago - https://github.com/voiceboys/sbcOS/tree/version_2.0

Thank you. How do I proceed from cloning the repo to having a functional/bootable iso. Apologies for the newbie question

**Comment by adubovikov** [comment]
https://github.com/voiceboys/sbcOS/blob/version_2.0/README.md

**Comment by airsay** [comment]
> https://github.com/voiceboys/sbcOS/blob/version_2.0/README.md

Thank you
